spec/OpenLineage.yml

```diff
@@ -0,0 +1,268 @@
+openapi: 3.0.2
+info:
+  title: OpenLineage
+  version: 0.0.1
+  description: OpenLineage is an open source **lineage and metadata collection API** for the data ecosystem.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /lineage:
+    post:
+      summary: Send an event related to the state of a run
+      description: Updates a run state for a job.
+      operationId: postRunStateUpdate
+      tags:
+        - OpenLineage
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RunStateUpdate'
+      responses:
+        '200':
+          description: OK
+
+components:
+  schemas:
+    RunStateUpdate:
+      type: object
+      properties:
+        transition:
+          description: the current transition of the run state. It is required to issue 1 START event and 1 of [ COMPLETE, ABORT, FAIL ] event per run. Additional events with no transition can be added to the same run. For example to send additional metadata after the run is complete
+          type: string
+          enum: [START, COMPLETE, ABORT, FAIL]
```

Comment:
How about a NONE transition and have transition be required, or not required with a default value of NONE. Since transition would be a queryable field (e.g. all events that are COMPLETE), it'd save us from having a placeholder field and doing an additional transformation on the server.